misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/TesterinaConstants.java

```diff
@@ -54,4 +54,12 @@
     public static final String REPORT_ZIP_NAME = REPORT_DIR_NAME + ".zip";
     public static final String REPORT_DATA_PLACEHOLDER = "__data__";
     public static final String FILE_PROTOCOL = "file://";
+
+    //Coverage dependencies
+    public static final String JACOCO_CORE_JAR = "org.jacoco.core-0.8.5.jar";
```

Comment:
Can't we package into the testerina-runtime jar since its the only usage?

Reply:
When I try to package it into the testerina-runtime.jar, it breaks the module dependency that testerina-core has on testerina-runtime. 

Originally, the jar files were packed into testerina-runtime.jar using `dist` in the build.gradle file :`dist 'org.jacoco:org.jacoco.core:0.8.5'`. When reverting to this and building the distribution, the 'testerina-core' module fails to pick up 'testerina-runtime' module as a dependency and fails to compile.